.env

```diff
@@ -0,0 +1,5 @@
+# This file is for default environment variables, and will be checked in to source code
```

Comment:
https://create-react-app.dev/docs/adding-custom-environment-variables. I have "CI" commented out because we really don't need it by default: any CI pipeline will automatically set this for us, and if not we would want to set it ourselves. But locally we are not in CI, so we want to avoid pretending we are.